### PR TITLE
Add conformance results for v1.19/minikube

### PR DESCRIPTION
**Pull request by azhao155**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [x] Did you include the product/project logo in SVG, EPS or AI format?  
* [x] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] If your product/project is open source, did you include the `repo_url`?
* [x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

File: v1.19/minikube/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+```
+
+Also you could try to verify minikube via deploying application and interact with cluster locally:
+https://minikube.sigs.k8s.io/docs/start/
+
+
+## Trigger the tests and get back the results
+
+We follow the [official instructions](https://github.com/cncf/k8s-conformance/blob/master/instructions.md):
+Run minikube with 2 nodes and containerd runtime.
```
Comment (medyagh) [inline v1.19/minikube/README.md:25]: add a note here that docker run-time is not confmrant and "containerd" runtime is needed.

**Review by medyagh — CHANGES_REQUESTED** [review]

**Comment by taylorwaggoner** [comment]
@azhao155 @medyagh is this ready for review or are more changes needed per the suggestions?

**Comment by azhao155** [comment]
@taylorwaggoner It's ready for review now. Thanks

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes

**Pull request merged**